daisy_workflows/build-publish/rhui/install_rhua.sh

```diff
@@ -89,6 +89,13 @@ for unit in rhui-health-check.{service,timer}; do
 done
 install -m 664 -t /etc/nginx/conf.d $tempdir/health_check.nginx.conf
 
+# Install RHUA sync status.
```

Comment:
this block is duplicating the one above including creating the dir; let's just amend line 86, only

Reply:
Updated.